rhel8/profiles/ospp.profile

```diff
@@ -1048,8 +1048,8 @@ selections:
     ## Enable rngd Service
     #################################################################
 
-    ## TO DO: https://github.com/ComplianceAsCode/content/issues/4472
-    #sysctl enable rngd.service
+    ## Enable the Hardware RNG Entropy Gatherer Service
+    - service_rngd_enabled
```

Comment:
This rule is already selected in master branch. Do you know why it is showing as a new addition? Below is the link to the master file pointing to the modification:
https://github.com/ComplianceAsCode/content/blob/777b143344da542fce58ab4bdbc1578a352845c1/rhel8/profiles/ospp.profile#L1054

Reply:
I don't know. Maybe I messed up the ospp.profile file. What I did in the end was that I took this file from the actual master and copied it into my branch and only added my rule at the end, see the last commit. How to fix it?

Reply:
No idea what happened here and I don't know how to fix. Maybe reverting this last commit and rebasing on top of master could help.